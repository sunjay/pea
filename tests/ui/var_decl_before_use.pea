fn main() {
    // Cannot use a variable before it is defined
    let b = d;
    // `a` is not defined until after its initializer has been walked
    let a = a;
}
