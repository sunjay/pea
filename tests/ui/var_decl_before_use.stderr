[tests/ui/var_decl_before_use.pea:3:13] error: cannot find name `d` in this scope

[tests/ui/var_decl_before_use.pea:5:13] error: cannot find name `a` in this scope

error: aborting due to 2 previous errors

